tests/elm/ElmLintBearTest.py

```diff
@@ -0,0 +1,145 @@
+from queue import Queue
+
+from bears.elm.ElmLintBear import ElmLintBear
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.settings.Section import Section
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coalib.testing.BearTestHelper import generate_skip_decorator
+from coala_utils.ContextManagers import prepare_file
+
+good_file_no_unexpected_comma = """
+ab =
+  "Hello World!"
+"""
+
+bad_file_unexpected_comma = """
+a, b =
+  "Hello World!"
+"""
+
+good_file_no_unexpected_end_of_input = """
+foo =
+  100
+"""
+
+bad_file_unexpected_end_of_input = """
+foo =
+"""
+
+good_file_not_empty = """
+add x y = x + y
+"""
+
+bad_file_empty = """
+"""
+
+good_file_comment = """
+foo =
+  5 * 3 - 8 / 1
+-- A single line comment
+"""
+
+bad_file_comment = """
+-- A single line comment
+"""
+
+good_file_list = """
+l =
+  ["one", "two", "three"]
+"""
+
+bad_file_list = """
+l =
+  ["one", "two", "three"
+"""
+
+good_file_function = """
+import Html exposing (text)
+four =
+  sqrt 16
+main =
+  text (toString [four])
+"""
+
+bad_file_function = """
+import Html exposing (text)
+four =
+  sqr 16
+main =
+  text (toString [four)
+"""
+
+result_message_bad_function = """\
+text (toString [four)\x1b[31m
```

Comment:
I am guessing the only way to do this is to set `TERM=dumb` before invoking haskell.
But that may not help as [hSupportsANSI](https://github.com/avh4/elm-format/search?utf8=%E2%9C%93&q=hSupportsANSI) isnt checked, and the [ANSI](https://github.com/avh4/elm-format/search?utf8=%E2%9C%93&q=ANSI) doesnt appear to otherwise care much about color, so we'd need to dig through the libraries to know -- or do a build.
Maybe later.